test/Ocean/SplitExplicit/simple_box_ivd.jl

```diff
@@ -209,30 +210,24 @@ function main(; restart = 0)
 
     if restart > 0
         direction = EveryDirection()
-        Q_3D, A_3D, t0 =
+        Q_3D, _, t0 =
             read_checkpoint(vtkpath, "baroclinic", ArrayType, mpicomm, restart)
-        Q_2D, A_2D, _ =
+        Q_2D, _, _ =
             read_checkpoint(vtkpath, "barotropic", ArrayType, mpicomm, restart)
```

Comment:
Eventually, the function `read_checkpoint` probably shouldn't return auxiliary fields at all, since they aren't needed, right?

Reply:
Yes. I could have tried to modify read_ & write_ checkpoint to allow this, but wanted to keep this set of changes internal to split-explicit. Wll be addressed in the (near) future.